### DISCUSSION TITLE
Homepage, can the port be changed?

### DISCUSSION THREAD

**Discussion by dheath73** [tteck/Proxmox#580] [Help Needed] [answered]
Firstly thank you so much for these scripts. I've just started using them and they are great.

I've just installed Homepage and I was wondering if it is possible to change the port to port 80 instead. The reason for me asking is so that I can just point my browser to the ip and it would then just work without needing to specify the port. At least that's how I believe it should work.

Any help would be greatly received.

Thanks again.

**Comment by tteck** [comment]
I don't know. Have you looked in `/opt/homepage/config/` for a config file that may be changed? The project is still pretty new, so documentation isn't there yet.

**Reply by dheath73** [reply]
Hello and thank you for your quick reply. I have looked at the setting.yaml and also the HomePage documentation however there was nothing there about amending the port.

I'm not familiar with next.js however I did some research. I followed an article that said I should add the port in the package.json file which is located in the /opt/homepage/ folder.

I adding -p 80 to each of the following lines like so and then rebooted. This did the trick, but I'm unsure if this will be overwritten with the next update? 

    "dev": "next dev -p 80",
    "build": "next build -p 80",
    "start": "next start -p 80",

Again thank you for your help and I hope this helps someone else.

**Reply by tteck** [reply]
Thank you for sharing!

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/580#discussioncomment-3777378

**Comment by erreib** [comment]
@dheath73 Thanks so much for this solution. I was also wondering if anyone had come up with a way to have the port change persist through updates?
